packages/react/src/index.ts

```diff
@@ -1,11 +1,14 @@
-import * as process from 'process';
-import { Buffer } from 'buffer';
+// Check if in browser environment
+if (typeof self !== 'undefined') {
```

Comment:
is this needed in both core and react?

Reply:
Yeah, since some FE's use some functions from /core and also /react uses /core.

Reply:
Removed from /react and tested 👌